lib/Config.lua

```diff
@@ -16,6 +16,10 @@
 local defaultConfig = {
 	-- Enables storage of `debug.traceback()` values on elements for debugging.
 	["elementTracing"] = false,
+	-- Enables instrumentation of shouldUpdate
+	["shouldUpdateInstrumentation"] = false,
+	-- Enables instrumentation of rendering
+	["renderInstrumentation"] = false,
```

Comment:
Do you think it's valuable to toggle instrumentation like this individually?

When working with a profiling debugger, I imagine we would either want all of the information logged, or none of it!

Reply:
I agree wholeheartedly, especially now that I'm faced with the idea of having my plugin behave accordingly if something is disabled. I'll make that change.